components/LevitatingLink.js

```diff
@@ -1,7 +1,8 @@
-export default function LevitatingLink ({ href, children }) {
+export default function LevitatingLink ({ href, children, handleClick, active }) {
```

Comment:
@nontachaiwebdev Yep, you may use `document.getElementById` and all the [data](https://github.com/reactbkk/2.0.0/blob/c1e5fcc6007fa7dafb68eeda3431574ce3b22334/pages/index.js#L15-L22) you need is already given in `this.props.navs` ^^